components/ILIAS/Chatroom/classes/gui/class.ilChatroomHistoryGUI.php

```diff
@@ -213,7 +213,10 @@ private function showMessages(
         $roomTpl->setVariable('PERIOD_FORM', $durationForm->getHTML());
 
         if ($room && $messages !== []) {
-            $this->mainTpl->addJavaScript('assets/js/socket.io.js');
+            $this->mainTpl->addJavaScript('assets/js/linkify.min.js');
+            $this->mainTpl->addJavaScript('assets/js/linkify-element.min.js');
+            $this->mainTpl->addJavaScript('assets/js/ilExtLink.js');
```

Comment:
Hi @lscharmer ,

please remove these calls and instead try to address adding the JS files in `\ilLinkifyUtil::initLinkify` (`\ilLinkifyUtil::getLocalJsPaths must be changed)`.

Best regards,
Michael

Reply:
Thx I didn't new this existed.